MAUI/DataGrid/Grouping.md

```diff
@@ -0,0 +1,572 @@
+        {
+            ColumnName = "OrderID",
+        });
+        dataGrid.GroupColumnDescriptions.Add(new GroupColumnDescription()
+        {
+            ColumnName = "ShipCity",
+        });
+        clearGroupingButton.Text = "Remove Grouping";
+        clearGroupingButton.TextColor = Colors.Black;
+        clearGroupingButton.BackgroundColor = Colors.White;
+        clearGroupingButton.Clicked += ClearGroupingButton_Click;
+        stackLayout.Children.Add(clearGroupingButton);
+        stackLayout.Children.Add(dataGrid);
+        this.Content = stackLayout;       
+    }
+}
+{% endhighlight%} 
+{% endtabs %}
+
+{% highlight c# %}
+private void ClearGroupingButton_Click(object sender, System.EventArgs e)
+{
+    //Clearing the Group
+    //dataGrid.GroupColumnDescriptions.Clear();
+
+    //Removing the Group based on group item
+    var groupColumn = dataGrid.GroupColumnDescriptions[1];
+    dataGrid.GroupColumnDescriptions.Remove(groupColumn);
+        
+    //Removing the Group based on group index
+    //dataGrid.GroupColumnDescriptions.RemoveAt(0);
+}
+{% endhighlight%} 
+
+Run the application to render the following output: 
+
+![Remove groups in runtime in a DataGrid](Images\Grouping\Before_RemovingGroup.png)
+![Remove groups in runtime in a DataGrid](Images\Grouping\After_RemovingGrouping.png)
+
+N> Clear or remove grouping on [GridTapped event](), [GridDoubleTapped event](), or [GridLongPressed event]().
+
+## Events
+
+### GroupExpanding event
+
+The [SfDataGrid.GroupExpanding]() event occurs when group is being expanded.
+ 
+The [GroupChangingEventArgs]() of the `GroupExpanding` event provides the information about the expanding group and it has the following members:
+
+ `Syncfusion.Data.Group`: Gets expanded group.
+
+ `Cancel`: Decides to cancel group expansion.
+ 
+Cancel group expansion by setting [GroupChangingEventArgs.Cancel]() to `true`.
+
+{% tabs %}
+{% highlight c# %}
+this.dataGrid.GroupExpanding += dataGrid_GroupExpanding;
+
+void dataGrid_GroupExpanding(object sender, Syncfusion.SfDataGrid.XForms.GroupChangingEventArgs e)
+{
+    if (e.Group.Key.Equals(1001))    
+        e.Cancel = true;    
+}       
+{% endhighlight %}
+{% endtabs %}
+
+### GroupExpanded event
+
+The [SfDataGrid.GroupExpanded]() event occurs after group is expanded.
+
+The [GroupChangedEventArgs]() of the `GroupExpanded` event provides the information about the expanded group and it has the following member:
+
+ `Syncfusion.Data.Group`: Gets the expanded group.
+
+### GroupCollapsing event 
+
+The [SfDataGrid.GroupCollapsing]() event occurs when group is being collapsed.
+
+The [GroupChangingEventArgs]()  of the `GroupCollapsing` event provides the information about the collapsing group and it contains the following member:
+
+ `Syncfusion.Data.Group`: Gets collapsed group.
+
+ `Cancel`: Decides to cancel the group collapsing.
+
+Cancel the group is being collapsed by using the [GroupChangingEventArgs.Cancel]() of `GroupCollapsing` event.
```

Comment:
Syncfusion.Data.Group: Gets the expanded group.
Cancel: Determines whether to cancel group expansion.

Cancel the group is being collapsed by setting GroupChangingEventArgs.Cancel to true.